src/diagnostics/LogPointsManager.ts

```diff
@@ -81,6 +82,11 @@ class DebugSessionManager {
         }
 
         const result: ISetLogpointResponse = await this._debugSession.customRequest("setLogpoint", { scriptId, lineNumber, columnNumber, expression });
+        if (result.error) {
+            vscode.window.showErrorMessage("Cannot set logpoint successfully. Please refer to DEBUG CONSOLE for details.");
+            appserviceUtil.getOutputChannel().appendLine(`Cannot set logpoint successfully, the error is ${result.error}.`);
```

Comment:
Do you want to show the output here? - in case the output window is not already opened.

Reply:
Yes. Added that.